app/code/Magento/Sales/Api/InvoiceCancelInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Sales\Api;
+
+/**
+ * Interface InvoiceCancelInterface
+ *
+ * @package Magento\Sales\Api
+ */
+interface InvoiceCancelInterface
```

Comment:
It may be a good idea to mark this with `@api`. This would also require providing PRs for other branches.